R-package/R/lgb.cv.R

```diff
@@ -134,7 +134,7 @@ lgb.cv <- function(params = list(),
     begin_iteration <- predictor$current_iter() + 1
   }
   # Check for number of rounds passed as parameter - in case there are multiple ones, take only the first one
-  n_trees <- c("num_iterations", "num_iteration", "n_iter", "num_tree", "num_trees", "num_round", "num_rounds", "num_boost_round", "n_estimators")
+  n_rounds <- .PARAMETER_ALIASES()[["num_iterations"]]
   if (any(names(params) %in% n_trees)) {
     end_iteration <- begin_iteration + params[[which(names(params) %in% n_trees)[1]]] - 1
```

Comment:
I guess it should be either `n_trees`, or `n_rounds`.

Reply:
I am bad at writing code haha

Thank you, good catch. As soon as I do #2335 CI will catch this for us.

Reply:
ok this is fixed